api/Clients.json

```diff
@@ -130,25 +130,17 @@
               "version_added": "17",
               "notes": "<code>Client</code> objects returned in most recent focus order."
             },
-            "firefox": [
-              {
-                "version_added": "54",
-                "notes": "<code>Client</code> objects returned in most recent focus order."
-              },
-              {
-                "version_added": "44",
-                "notes": "<a href='https://www.mozilla.org/en-US/firefox/organizations/'>Extended Support Releases (ESR)</a> before Firefox 78 ESR do not support service workers and the Push API."
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "54",
-                "notes": "<code>Client</code> objects returned in most recent focus order."
-              },
-              {
-                "version_added": "44"
-              }
-            ],
+            "firefox": {
+              "version_added": "44",
+              "notes": [
+                "Since Firefox 54, <code>Client</code> objects are returned in most recent focus order.",
```

Comment:
Is this the per-spec behavior or not? Presumably there was a bug before 54, and we should have a note or `partial_implementation` for that?